OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/ViewKeyAdvShareFragment.java

```diff
@@ -455,8 +455,17 @@ protected void onPostExecute(Bitmap qrCode) {
     }
 
     private void uploadToKeyserver() {
+        long keyId = 0;
```

Comment:
Correct. In this case, the keyId variable doesn't have to be initialized - if the compiler complains about that, it means the variable might be used when there's a 0 in there, which is a sign that something is awry.

Apart from that, we use snackbars instead of toasts